recipes/python-simplemachinesforum/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "simplemachinesforum" %}
+{% set version = "0.4.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/simplemachinesforum-{{ version }}.tar.gz
+  sha256: 89699e282621b9c53ccf333149ec5a6fa51a101632a24eff69952cf2b320c5c5
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.0
+  run:
+    - python >=3.0
+    - requests
+    - bs4
+
+test:
+  imports:
+    - simplemachinesforum
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://about.me/oliver-zehentleitner
+  summary: Python request API to simplemachinesforum
+  description: |
+    # python-simplemachinesforum
+    Python request API to Simple Machines Forum: https://www.simplemachines.org/
+
+    ## Important
+    A forum is for humans, please use this piece of software only in combination with YOUR OWN FORUM or by order of the forum owner itself to provide a valuable service. Be aware that bots are not tolerated on most boards and its use will lead to a ban.
```

Comment:
This is not a blocker but lines shouldn't be longer than 80-100 characters and not go outside of the view screen.

Please change that with the next update.

Reply:
solved